### DISCUSSION TITLE
requirements not met on 22h2

### DISCUSSION THREAD

**Discussion by FutherHD** [Atlas-OS/Atlas#1241] [Help & Support]
Hello, I'm trying to switch to atlas os but it's saying that my build isnt supported, I'm trying to use the lastest atlas version which is compatible with 23h

**Comment by Ast3risk-ops** [comment]
Is this windows 10 or 11, if it's 11 update to 23h2.

**Reply by FutherHD** [reply]
Hey thanks for the answer. It's windows 11 and I don't know why but windows don't suggest the 23h2 but it normally should work with the 22h2 right? or should I use another atlas version?

**Reply by Ast3risk-ops** [reply]
> Hey thanks for the answer. It's windows 11 and I don't know why but windows don't suggest the 23h2 but it normally should work with the 22h2 right? or should I use another atlas version?

It doesn't support 11 22h2. Don't use an older atlas version.

**Reply by FutherHD** [reply]
I'm going to install 23h2 from the iso, thanks for the infos
